.github/workflows/rust.yml

```diff
@@ -70,6 +70,24 @@ jobs:
         run: |
           rustup toolchain install stable --no-self-update
           rustup default stable
+      - name: Install Protoc
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v28.3/protoc-28.3-linux-x86_64.zip
+          unzip protoc-28.3-linux-x86_64.zip -d $HOME/.local
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
+      - name: Install Protoc
+        if: matrix.os == 'macos-13'
+        run: |
+          curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v28.3/protoc-28.3-osx-x86_64.zip
+          unzip protoc-28.3-osx-x86_64.zip -d $HOME/.local
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
+      - name: Install Protoc
+        if: matrix.os == 'macos-latest'
+        run: |
+          curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v28.3/protoc-28.3-osx-aarch_64.zip
+          unzip protoc-28.3-osx-aarch_64.zip -d $HOME/.local
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
```

Comment:
```suggestion
      - name: Install Protoc
        if: runner.os == 'macOS'
        run: |
          curl -LO "https://github.com/protocolbuffers/protobuf/releases/download/v28.3/protoc-28.3-osx-$(uname -m).zip"
          unzip protoc-28.3-osx-x86_64.zip -d $HOME/.local
          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
```

Reply:
thanks. Can you swap `x86_64` on line 83 with uname as well?

Reply:
Sorry I missed that. In that case, it might be simpler to rename the file with curl's `-o` option.